### PR TITLE
chore: return device that is running out of disk

### LINKED ISSUE #7215: Not meaningful log about disk capacity
**Is this a BUG REPORT or FEATURE REQUEST?**:

 /kind enhancement

**What happened**:

```
{"component":"virt-launcher","level":"error","msg":"No disk capacity","pos":"manager.go:661","timestamp":"2022-02-09T22:18:22.107956Z"}
{"component":"virt-launcher","level":"error","msg":"No disk capacity","pos":"manager.go:661","timestamp":"2022-02-09T22:18:22.109726Z"}
```

**What you expected to happen**:

I would expect to at least know which disk is full and why do we care.

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- KubeVirt version (use `virtctl version`):
- Kubernetes version (use `kubectl version`):
- VM or VMI specifications:
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

### PR DESCRIPTION
**Pull request by Gre8t**
**What this PR does / why we need it**:
It returns the device that is running out of disk

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #7215

**Special notes for your reviewer**:

**Release note**:

```release-note

```

### CODE REVIEW

File: pkg/virt-launcher/virtwrap/manager.go [hunk 1]
```diff
@@ -681,7 +681,7 @@ func expandDiskImageOffline(imagePath string, size int64) error {
 
 func possibleGuestSize(disk api.Disk) (int64, bool) {
 	if disk.Capacity == nil {
-		log.DefaultLogger().Error("No disk capacity")
+		log.DefaultLogger().Errorf("No disk capacity on %v", disk.Device)
```
Comment (xpivarc) [inline pkg/virt-launcher/virtwrap/manager.go:684]: A more helpful information would be the source. (To be more precise the path to the file or device)

Comment (Gre8t) [reply pkg/virt-launcher/virtwrap/manager.go:684]: @xpivarc how do I test this changes? I can start the build server on my machine, but I can't figure out how to test just the changes I have made to that unit of code.

Comment (alicefr) [reply pkg/virt-launcher/virtwrap/manager.go]: I think the simplest way is to write a small unit test in manager_test.go. IIRC, it should be able to use the local function.

Comment (Gre8t) [reply pkg/virt-launcher/virtwrap/manager.go]: Writing a small unit test for any changes you have demands you have a good grasp of the various kubevirt packages(maybe there is a trick to do it, and I just suck at it).

Comment (alicefr) [reply pkg/virt-launcher/virtwrap/manager.go]: well, you asked a way how to check your change. First, I don't think we need the test for upstream, it is a very simple change for a logging message. However, if you want to check it, you could add a unit test  in [manager_test.go](https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-launcher/virtwrap/manager_test.go)  and call the function and pass a disk with empty `Capacity`. If you focus on that test, you should see login message

**Comment by xpivarc** [comment]
@stu-gott Does the disk always have a backing file?

**Pull request closed without merging**